Makefile.include

```diff
@@ -530,7 +531,8 @@ endif
 
 # Generate machine readable RIOT VERSION macro
 RIOT_VERSION_CODE ?= $(shell echo ${RIOT_VERSION} | \
-                      sed -E 's/([0-9]+).([0-9]+).?([0-9]+)?.*/RIOT_VERSION_NUM\\\(\1,\2,0\3,${RIOT_EXTRAVERSION}\\\)/')
+                      sed -E 's/([0-9]+).([0-9]+).?([0-9]+)?.*/RIOT_VERSION_NUM\\\(\1,\2,0\3,${RIOT_EXTRAVERSION}\\\)/' | \
+                      grep RIOT_VERSION_NUM && echo -n "" || echo "${RIOT_VERSION_DUMMY_CODE}")
```

Comment:
Why the `&& echo -n ""`?

Reply:
It's a bit of a hack, just a no-op so I get an else branch when `grep` returns with a non-0 exit code.

Reply:
But you do not need to have a no-op. `grep` will return the matching line (which is the only one printed) just fine and have a 0 exit code...

Reply:
Yea I somehow thought `||` only worked if there was a `&&` before, but that's not the case.